src/services/services.ts

```diff
@@ -1958,6 +1959,12 @@ namespace ts {
                     (char >= CharacterCodes.A && char <= CharacterCodes.Z) ||
                     (char >= CharacterCodes._0 && char <= CharacterCodes._9);
             }
+
+            function isNodeModulesFile(path: string): boolean {
+                const node_modulesFolderName = "/node_modules/";

```

Comment:
What if the whole project is situated nested inside node_modules somewhere? Will all TODOs suddenly disappear?

/myDir1/node_modules/myOtherDirectory/**myProjectRoot**/src/blablaFile.ts

People do put temporary junk in their node_modules for the purpose of quick-checking someething and avoiding erroneous commit.

Additionally, in complex large projects there may be a legitimate need to keep projects in node_modules due to some process optimisation.

All in all, the check should only turn true if node_modules appear after the project root in full path, not before it.